.github/workflows/main.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    container:
+      image: themattrix/tox
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.0.0
+    - name: Tests
+      run: |
+          apt-get update -yqq
+          apt-get install shellcheck -yqq
```

Comment:
IMO it is bad to have `-qq` here – in case of some problems, it would be nice to know what was wrong on `apt-get` side.

Reply:
You're right. Done.